config/crd/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ resources:
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-- patches/webhook_in_integrationtestscenarios.yaml
+#- patches/webhook_in_integrationtestscenarios.yaml
```

Comment:
Was this patch only for conversion webhooks or will we still need it for the validation webhook?

Reply:
Yes, config/crd/patches/webhook_in_integrationtestscenarios.yaml contained only conversion. This change wont touch validation.